src/neo-vm/Utility.cs

```diff
@@ -25,6 +25,26 @@ static Utility()
             StrictUTF8.EncoderFallback = EncoderFallback.ExceptionFallback;
         }
 
+        public static BigInteger ModInverse(this BigInteger n, BigInteger modulus)
```

Comment:
Do you think we need to check some limits to "n" and "modulus" here at Utility?
At this point, "n" could surpass 2^256, although there may not be an use-case for it.

Reply:
In neo-vm, all integers are limited in 32 bytes.

Reply:
Ok, although this is Utility, and someone else using NeoVM library could surpass this value (as BigInteger in C# in unbounded). Anyway, this is not a risk, just mentioning that it could happen.